fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -451,8 +454,16 @@ public TGetTaskInfoResult getTasks(TGetTasksParams params) throws TException {
             TTaskInfo info = new TTaskInfo();
             info.setTask_name(task.getName());
             info.setCreate_time(task.getCreateTime() / 1000);
-            // Now there are only MANUAL types of Tasks
-            info.setSchedule("MANUAL");
+            String scheduleStr = task.getType().name();
+            if (task.getType() == Constants.TaskType.PERIODICAL) {
+                TaskSchedule schedule = task.getSchedule();
+                scheduleStr += " (START "
```

Comment:
1. move this logic in `TaskSchedule.describe()`
2. use StringBuffer to append, not use String operator+

Reply:
> 1. move this logic in `TaskSchedule.describe()`
> 2. use StringBuffer to append, not use String operator+

Why not use StringBuilder?  Initializing StringBuilder in a method will not have thread safety issues

Reply:
move some logic into `TaskSchedule toString()`.